.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+before_script: npm run-script build
+node_js:
+  - 0.10
```

Comment:
@iamakulov maybe it should be just `node` and that's it?

> Specifying node will run using the latest stable Node.js release

We don't really need to test code in nodejs, it's used only for build and tests. So seems it doesn't matter, right?